samples/java_springboot/15.handling-attachments/src/main/java/com/microsoft/bot/sample/attachments/AttachmentsBot.java

```diff
@@ -255,7 +255,12 @@ private static Attachment getInternetAttachment() {
     }
 
     private CompletableFuture<byte[]> getFileData(String filename) {
-        try (InputStream inputStream = Thread.currentThread().getContextClassLoader().getResourceAsStream(filename)) {
+        Attachment adaptiveCardAttachment = new Attachment();
+
+        try (
+            InputStream inputStream = adaptiveCardAttachment.getClass()
+                .getClassLoader().getResourceAsStream(filename)
+        ) {
```

Comment:
Right here we are creating an attachment that is not used for anything other than getting its class loader. In that case, we should either keep this untouched if it works or find an alternative way of getting the correct class loader.